guide/how-it-works/ecash/cashu.md

```diff
@@ -0,0 +1,116 @@
+
+https://www.figma.com/community/file/888680264445459448
+https://www.figma.com/community/file/995256542920917246/BDG---Private-key-management-illustrations
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/ecash.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/ecash@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/ecash-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/ecash-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Cashu
+Cashu is an ecash protocol on that interoperates with the Lightning Network. Unlike Fedimint, which uses a federated model to distribute trust among multiple entities, Cashu operates with independent mints, offering greater flexibility and control to users. This makes Cashu distinct in providing an easy to deploy, flexible, and agile approach to managing Ecash on bitcoin.
+
+{% include tip/open.html color="blue" icon="info" label="User Experience Tip" %}
+
+When designing for Cashu, consider providing options for users to set default mints and auto-swap preferences. Since Cashu mints are independently operated, users may prefer to default to a trusted mint.
+
+{% include tip/close.html %}
+
+## Backup and Restore for Cashu
+The Cashu backup and restoration process is designed to ensure users can securely recover their wallets and maintain access to their ecash tokens, even if they switch devices or experience data loss.
+
+{% include tip/open.html color="blue" icon="info" label="Single Use Recovery" %}
+
+Only use your recovery seed phrase once. Repeated use of the seed phrase for restoration can lead to synchronization issues and potential errors. This is because each time you restore, you might be dealing with an outdated state of your wallet, which can cause discrepancies in token balances and transactions.
+
+{% include tip/close.html %}
+
+
+### Backup Process
+1. Deterministic Wallet with Seed Phrase: Cashu uses a deterministic wallet model, where all cryptographic keys and tokens can be derived from a single seed phrase. This seed phrase is generated when the wallet is first created.
+
+2. Secure Storage: Users are advised to store their seed phrase securely. It is crucial to keep this seed phrase in a safe, physical format (such as written on paper) and stored in a secure location to prevent unauthorized access or loss. The best practices in the [manual backup]({{ '/guide/private-key-management/manual-backup/' | relative_url }}) section for Lightning and on-chain wallets also apply to Cashu wallets. 
+
+{% include tip/open.html color="blue" icon="info" label="Mint Information Needed" %}
+
+Keep a record of the mints you are connected to. During the restoration process, you will need to reconnect to these specific mints to re-verify your tokens. Without this information, you might not be able to fully restore your wallet or verify your tokens.
```

Comment:
This is a great suggestion. I like the idea of automatically encrypting a backup of the mint information and having the app scan for the backup during the restoration process. If it finds the backup, it should ask the user to confirm. If it doesn’t find a backup, it should proceed with the manual mint entry flow. This is a great idea, but is it too biased to include in ‘best practices’? Or is it too much to suggest that wallets should implement this?

Reply:
In step 2 of the backup process I updated the link from the manual backup section to the [backup & recovery ](https://bitcoin.design/guide/daily-spending-wallet/backup-and-recovery/landing-page/) section of the guide. Since automatic cloud backup is referenced in this section I think we have this covered. I think referencing the backup & recovery section and suggestion that the user follow the best practices in that guide covers this.